data/json/items/containers/containers.json

```diff
@@ -2595,410 +2657,410 @@
-        "weight": 5
+        "weight": 1
       },
       {
         "id": "bird_case",
         "name": { "str": "Bird Box smartphone case" },
         "description": "This waterproof smartphone case features a graphic of a blindfolded face on a black background with subtle patterns in the wallpaper.  The sidewalls have a broken, abstract hue with some light elements and waves.",
-        "weight": 5
+        "weight": 1
       },
       {
         "id": "dracula_case",
         "name": { "str": "Dracula smartphone case" },
         "description": "This waterproof smartphone case features an image of a classic gothic castle with a sinister font in tones of blood red, deep black, and gothic elements.  The sidewalls resemble the stone-walled interiors of the same castle, with colors matching the dominant colors of the case.",
-        "weight": 5
+        "weight": 1
       },
       {
         "id": "darkness_case",
         "name": { "str": "Darkness Falls smartphone case" },
         "description": "This waterproof smartphone case features an image of a creepy tooth fairy with sharp, acidic white accents on a dark black background.  The sidewalls include personal toothbrush elements.",
-        "weight": 5
+        "weight": 1
       },
       {
         "id": "mist_case",
         "name": { "str": "The Mist smartphone case" },
         "description": "This waterproof smartphone case features a graphic of a white, ominous cloud over a suburban environment at night with tree branches and lightning in the background.  The sidewalls include misty abstract elements and tree graphics.",
-        "weight": 5
+        "weight": 1
       },
       {
         "id": "pyramidhead_case",
         "name": { "str": "Silent Hill smartphone case" },
         "description": "This waterproof smartphone case features a graphic of a sinister fog with an iconic pyramid-headed silhouette in a dark world.  The sidewalls evoke an intermittent window pattern with the world's colors moving over it.",
-        "weight": 5
+        "weight": 1
       },
       {
         "id": "slender_case",
         "name": { "str": "Slenderman smartphone case" },
         "description": "This waterproof smartphone case features a graphic of black and white mixed with static running through a dark background with fog and trees.  You can see the mysterious figure of the pale, faceless stalker among those trees.  The sidewalls resemble trees, with the same static running through them.",
-        "weight": 5
+        "weight": 1
       },
       {
         "id": "bates_case",
         "name": { "str": "Bates Motel smartphone case" },
         "description": "This waterproof smartphone case features a graphic of a creepy motel sign that can become warm and soothing with a touch of nostalgia for the old times.  The sidewalls feature floral patterns serenading the hotel's name.",
-        "weight": 5
+        "weight": 1
```

Comment:
```suggestion
```
Ignoring that these changes aren't related to the PR, weight 1 is the default so they shouldn't be explicitly defined as such